utils/spacewalk-common-channels.ini

```diff
@@ -3179,3 +3179,50 @@ gpgkey_url = http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
 gpgkey_id = 2F86D6A1
 gpgkey_fingerprint = 94E2 79EB 8D8F 25B2 1810 ADF1 21EA 45AB 2F86 D6A1
 repo_url = http://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=%(arch)s
+
+[openeuler2203]
+archs	= x86_64, aarch64
+name	= openEuler 22.03 %(arch)s
+gpgkey_url = https://repo.openeuler.org/openEuler-22.03-LTS/OS/x86_64/RPM-GPG-KEY-openEuler
+gpgkey_id = AC26B6D3
+gpgkey_fingerprint = 12EA 74AC 9DF4 8D46 C69C  A0BE D557 065E B25E 7F66
+repo_url = https://repo.openeuler.org/openEuler-22.03-LTS/OS/%(arch)s
+
+[openeuler2203-update]
+archs	= x86_64, aarch64
+name	= openEuler 22.03 Update %(arch)s
+label	= openeuler2203-update-%(arch)s
+basechannels = openeuler2203-%(arch)s
+gpgkey_url = https://repo.openeuler.org/openEuler-22.03-LTS/OS/x86_64/RPM-GPG-KEY-openEuler
+gpgkey_id = AC26B6D3
+gpgkey_fingerprint = 12EA 74AC 9DF4 8D46 C69C  A0BE D557 065E B25E 7F66
+repo_url = https://repo.openeuler.org/openEuler-22.03-LTS/update/%(arch)s
+
+[openeuler2203-everything]
+archs	= x86_64, aarch64
+name	= openEuler 22.03 Everything %(arch)s
+label	= openeuler2203-everything-%(arch)s
+basechannels = openeuler2203-%(arch)s
+gpgkey_url = https://repo.openeuler.org/openEuler-22.03-LTS/everything/x86_64/RPM-GPG-KEY-openEuler
+gpgkey_id = AC26B6D3
+gpgkey_fingerprint = 12EA 74AC 9DF4 8D46 C69C  A0BE D557 065E B25E 7F66
+repo_url = https://repo.openeuler.org/openEuler-22.03-LTS/everything/%(arch)s
+
+[openeuler2203-epol]
+archs	= x86_64, aarch64
+name	= openEuler 22.03 EPOL %(arch)s
+label	= openeuler2203-epol-%(arch)s
+basechannels = openeuler2203-%(arch)s
+gpgkey_url = https://repo.openeuler.org/openEuler-22.03-LTS/OS/x86_64/RPM-GPG-KEY-openEuler
+gpgkey_id = AC26B6D3
+gpgkey_fingerprint = 12EA 74AC 9DF4 8D46 C69C  A0BE D557 065E B25E 7F66
+repo_url = https://repo.openeuler.org/openEuler-22.03-LTS/EPOL/%(arch)s
+
+[openeuler2203-uyuni-client]
```

Comment:
Remember to add `openeuler2203-uyuni-client-devel` later.

Reply:
Done